### ISSUE TITLE
NTR: Anti-Jo-1-antibody positivity

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5797]
**Preferred term label:**
Anti-Jo-1-antibody positivity

**Synonyms**

**Definition (free text, please give PubMed ID)**
The presence of autoantibodies in the serum that react to the histidyl-tRNA-synthetase.

Comment: Histidyl-tRNA synthetase (HRS = Jo-1) represents a key autoantibody target in the anti-synthetase syndrome that is marked by myositis as well as extra-muscular organ complications including interstitial lung disease (ILD).

PMID: 26210509
DOI:10.1007/s11926-015-0532-1

**Parent term (use hpo.jax.org/app)**
Autoimmune antibody positivity HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Antisynthetase Syndrome ORPHA:81

**Your nano-attribution (ORCID)**
lmu: chrapp, @mgriese, @katrinknoflach
